src/Http/Middleware/IdpMiddleware.php

```diff
@@ -69,6 +66,11 @@ public function handle(Request $request, Closure $next)
                 $this->addExtraParametersToUser($user);
 
                 Auth::setUser($user);
+
+                if ($request->query('token')) {
+                    $request->query->remove('token');
+                    return redirect($request->url());
```

Comment:
Dalla documentazione mi pare di capire che il metodo `url()` ritorna già l'URL senza la query string.. Quindi il remove precedente sarebbe inutile. Però non mi torna, perché se ci sono altri parametri in qs non li possiamo perdere.